docs/example_nodejs.md

```diff
@@ -1,34 +1,314 @@
+
 ---
 title: "Example: NodeJS"
 layout: docs
 ---
 
-You're a NodeJS developer. You want to set up Tilt with your project.
+The best indicator of a healthy development workflow is a short feedback loop.
+
+Kubernetes is a huge wrench in the works.
+
+Let's fix this.
+
+In this example, we're going to take you through a simple “hello world” server written in NodeJS that uses [Mustache](https://mustache.github.io/) for templating and [Express](https://expressjs.com/) for serving.
+
+We'll use Tilt to:
+
+- Run the server on Kubernetes
+- Measure the time from a code change to a new process
+- Optimize that time for fast feedback
+
+This particular example server doesn't do much, but it's useful to confirm that Tilt is working as expected in your environment.
+
+All the code is in this repo:
+
+[tilt-example-nodejs](https://github.com/windmilleng/tilt-example-nodejs)
+
+To skip straight to the fully optimized setup, go to this subdirectory:
+
+[Recommended Tiltfile](https://github.com/windmilleng/tilt-example-nodejs/tree/master/3-recommended)
+
+_Quick note: Tilt's first deployment of a service takes a few seconds longer than subsequent ones, due to some behind-the-scenes setup. Measurements in this guide focus on non-initial builds._
```

Comment:
A first-time Tilt user isn't going to understand or even care about this comment, let's either move this to a footnote or axe it entirely